src/app/static/src/app/localStorageManager.ts

```diff
@@ -47,6 +48,10 @@ export class LocalStorageManager {
             localStorage.clear();
             localStorage.setItem("user", currentUser);
         }
+        if (currentMode != window.localStorage.getItem("mode")) {
```

Comment:
Or perhaps better to modify all this logic to save/retrieve app state under a different key for the data exploration app

Reply:
Yeah, agree that's better. And yes we can.